src/ResourceManager/Resources/Commands.Resources/RoleDefinitions/GetAzureRoleDefinitionCommand.cs

```diff
@@ -54,15 +54,25 @@ public override void ExecuteCmdlet()
                 Scope = Scope,
                 ResourceIdentifier = new ResourceIdentifier
                 {
-                    Subscription = DefaultProfile.DefaultContext.Subscription.Id.ToString()
+                    Subscription = (DefaultProfile.DefaultContext.Subscription.Id == null)?null:DefaultProfile.DefaultContext.Subscription.Id.ToString()
```

Comment:
```suggestion
                    Subscription = DefaultProfile.DefaultContext.Subscription.Id?.ToString()
```
Just use the [null conditional operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators) instead.